tests/test_spec_collectors.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test that spec collectors is ok"""
+# pylint: disable=redefined-outer-name,unused-argument
+import os.path
+from unittest import mock
+
+import pytest
+import toml
+
+from pytest_allure_spec_coverage.models.collector import Collector
+from pytest_allure_spec_coverage.models.scenario import Scenario, Parent
+from pytest_allure_spec_coverage.spec_collectors import SphinxCollector
+
+SCENARIOS = [
+    Scenario(
+        name="simple_scenario",
+        display_name="Simple scenario",
+        parents=[Parent(name="scenarios", display_name="There is some scenarios")],
+        link=None,
+        branch=None,
+    ),
+    Scenario(
+        name="scenario_with_parent",
+        display_name="Scenario with parent",
+        parents=[
+            Parent(name="scenarios", display_name="There is some scenarios"),
+            Parent(name="parent_folder", display_name="Parent folder for scenarios"),
+        ],
+        link=None,
+        branch=None,
+    ),
+    Scenario(
+        name="scenario_with_parents",
+        display_name="Scenario with two level of parents",
+        parents=[
+            Parent(name="scenarios", display_name="There is some scenarios"),
+            Parent(name="parent_folder", display_name="Parent folder for scenarios"),
+            Parent(name="second_level_parent", display_name="Second level parent folder"),
+        ],
+        link=None,
+        branch=None,
+    ),
+]
+
+
+class TestCollector(Collector):
+    """Simple collector for config test"""
+
+    def collect(self):
+        pass
+
+    def validate_config(self):
+        pass
+
+
+@pytest.fixture()
+def pyproject_toml(request):
+    """pyproject content"""
+    return {"tool": {"pytest_allure_spec_coverage": request.param}}
```

Comment:
Pls add a check, that fixture is parametrized
Like 
```
if not hasattr(request, "param"):
  raise ValueError("bla bla bla")
```

Reply:
fix